### PR TITLE
feat: Update Colab for Rapid Eval SDK GA

### PR DESCRIPTION
**Pull request by coolalexzb**
# Description

Update colab notebooks for model based metrics update in Rapid Eval SDK GA

gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb
gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb
gemini/evaluation/gemini_function_calling_evaluation_rapid_evaluation_sdk.ipynb
gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb

### CODE REVIEW

File: gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb [hunk 1]
```diff
@@ -1,913 +1,900 @@
 {
```
Comment (jsondai) [inline gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb:1]: <p>Rapid Eval SDK GA&nbsp; ==&gt; "Gen AI Eval Service SDK".  Throughout. </p><p> </p><p>Please update the hyperlinks if you changed the filename.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jsondai) [inline gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.    import plotly.graph_objects as go
</pre><p>This one can go to "General"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

File: gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb [hunk 2]
```diff
@@ -1,796 +1,675 @@
 {
```
Comment (jsondai) [inline gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <p>"Gen AI Eval Service SDK"&nbsp;</p><p> </p><p>If we change the filename, please also update the hyperlinks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jsondai) [inline gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <p>Remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (jsondai) [inline gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.    LOCATION = os.environ.get("GOOGLE_CLOUD_REGION", "us-central1")
</pre><p>LOCATION = "us-central1"  # @param {type:"string"}</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (jsondai) [inline gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.    from tqdm import tqdm
</pre><p>Let's remove this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (jsondai) [inline gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    for _, (model_name, model) in tqdm(
</pre><p>Nested tqdm progress bars cause display errors, we can remove the tqdm from this for-loop.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

File: gemini/evaluation/gemini_function_calling_evaluation_rapid_evaluation_sdk.ipynb [hunk 3]
```diff
@@ -1,872 +1,728 @@
 {
```
Comment (jsondai) [inline gemini/evaluation/gemini_function_calling_evaluation_rapid_evaluation_sdk.ipynb:1]: <p>"Gen AI Eval Service SDK"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb [hunk 4]
```diff
@@ -1,842 +1,803 @@
 {
```
Comment (jsondai) [inline gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb:1]: <p>"Gen AI Eval Service SDK'</p><p> </p><p>We also need to rename this tutorial to "Compare models". After changing filenames, please update the hyperlinks above </p><p> </p><p>See http://shortn/_SbzHwiTACn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jsondai) [inline gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb:1]: <p>Gen AI Eval Service SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (jsondai) [inline gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb:1]: <p>ditto</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (jsondai) [inline gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb:1]: <p>Naming of the section title and model names. use "candidate model" "baseline model", "Gemini-1.0-Pro", also "Gen AI Eval Service SDK"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (jsondai) [inline gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Gemini Pro
</pre><p>remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (jsondai) [inline gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #22.    # PaLM 2 Text Bison
</pre><p>remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (jsondai) [inline gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    for _, (model_name, model) in tqdm(
</pre><p>remove nested tqdm</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Review by holtskinner — APPROVED** [review]

File: gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb (hunk 1 above)
Comment (coolalexzb) [reply gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb]: <p>Done</p>

Comment (coolalexzb) [reply gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb]: <p>This is fix by lint to make the import order is correct. Remove #General and #Main comment</p>

File: gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb (hunk 2 above)
Comment (coolalexzb) [reply gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <p>done</p>

Comment (coolalexzb) [reply gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <p>Done.</p>

Comment (coolalexzb) [reply gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <p>Done</p>

Comment (coolalexzb) [reply gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <p>Done.</p>

Comment (coolalexzb) [reply gemini/evaluation/evaluate_gemini_model_settings_rapid_evaluation_sdk.ipynb:1]: <p>Done</p>

File: gemini/evaluation/gemini_function_calling_evaluation_rapid_evaluation_sdk.ipynb (hunk 3 above)
Comment (coolalexzb) [reply gemini/evaluation/gemini_function_calling_evaluation_rapid_evaluation_sdk.ipynb:1]: <p>Done.</p>

File: gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb (hunk 4 above)
Comment (coolalexzb) [reply gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb]: <p>Done</p>

Comment (coolalexzb) [reply gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb]: Done.

Comment (coolalexzb) [reply gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb]: Done.

Comment (coolalexzb) [reply gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb]: Done

Comment (coolalexzb) [reply gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb]: This is not autosxs example. not update to candidate model and baseline model

Comment (coolalexzb) [reply gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb]: Done.

**Pull request merged**